portal-web/test/functional/com/liferay/portalweb/macros/analyticscloud/ACSettings.macro

```diff
@@ -20,6 +20,18 @@ definition {
 		Click(locator1 = "ACSettings#KEYWORD_SEND_BUTTON");
 	}
 
+	macro assertDataSource {
+		ACSettings.goToDataSources();
+
+		Click(locator1 = "ACDXPSettings#AC_GENERIC_ADD_OR_CONFIRMATION_BUTTON");
```

Comment:
Hi, @AfonsoHSantos. If this macro is intended to be assert then I think it's best not to put a navigation command in it so it doesn't cause problems for other people who might want to use it.

Put the commands in testcase and create a macro to put the `Click(locator1 = "ACDXPSettings#AC_GENERIC_ADD_OR_CONFIRMATION_BUTTON");`, remember to see where that `click` is used to substitute your new macro